### PR TITLE
feat: dev card v2

### PR DESCRIPTION
**Pull request by sshanzel**
NOTE: you might see a lot of explicit usage of primitive colors, that was intended since the bg doesn't change based on the app settings' theme.

## Changes
- Introduce devcard that changes its display based on props.
- Applied the devcard on the tooltip.

~~TODO: switch the query to utilize Milan's API.~~

Preview: https://dailydotdev.slack.com/archives/C06FDFQC6QZ/p1707325533001429
Screenshot 2024-02-08 at 11 09 40 AM
Screenshot 2024-02-08 at 11 09 23 AM
Screenshot 2024-02-08 at 11 09 04 AM

## Events

Did you introduce any new tracking events?
Don't forget to update the [Analytics Taxonomy sheet](https://docs.google.com/spreadsheets/d/18Lv7zXges9QfVX5VYL1a-Hyl0e1sQ3sLr0OK8YZWKXI/edit#gid=0)

Log the new/changed events below:

| Type   | event_name  | value |
|--------|-------------|-------|
| Change/New | event name  | extra: { ... } |

### On those affected packages:
- [ ] Have you done sanity checks in the webapp?
- [ ] Have you done sanity checks in the extension?
- [ ] Does this not break anything in companion?

### Did you test the modified components media queries?
- [ ] MobileL (420px)
- [ ] Tablet (656px)
- [ ] Laptop (1020px)

#### Did you test on actual mobile devices?
- [ ] iOS (Chrome and Safari)
- [ ] Android

MI-150 #done

### CODE REVIEW

File: packages/eslint-rules/lib/rules/no-custom-color.js [hunk 1]
```diff
@@ -20,6 +20,7 @@ const alwaysValidClasses = [
   'bg-no-repeat',
   'bg-center',
   'bg-contain',
+  'text-opacity-64', // refactor in a separate PR to something that would filter all text-opacity-*
```
Comment (kopancek) [inline packages/eslint-rules/lib/rules/no-custom-color.js:23]: Out of scope, but looking at this, should we add a ticket to rename the rule to `no-custom-classes.js`? 🤔

**Review by kopancek — APPROVED** [review]
LGTM! :shipit:

File: packages/eslint-rules/lib/rules/no-custom-color.js (hunk 1 above)
Comment (sshanzel) [reply packages/eslint-rules/lib/rules/no-custom-color.js:23]: Agreed. Will write one for this.

**Review by kopancek — APPROVED** [review]

File: packages/shared/src/lib/image.ts [hunk 2]
```diff
@@ -159,6 +159,10 @@ export const cloudinary = {
       'https://daily-now-res.cloudinary.com/image/upload/v1705386465/Splash_v1lxjk.svg',
     fire: 'https://daily-now-res.cloudinary.com/image/upload/v1705386465/Hot_nrqvv5.svg',
   },
+  devcard: {
+    defaultCoverImage:
+      'https://s3-alpha-sig.figma.com/img/49ce/0824/73cb55195cba281ff2aaefde4a83f88d?Expires=1708300800&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=ML5EVkJLLpkNFv327CTitgGzDCTc5tj8Ob5yLYlQoVRy6yDeyy-TRs8uJSocZfuQWZbZ~YE9boQIU8paA86KZR01fZDl5VukPG~W0eAUmBtjM~VeDT1eP0EPSng35MpUdOLeh970kyOAyilaQwj~JzOrieE91IadbqTnZrED4CjWCpUsfzS33zU5uw9kljcPEbHEEuKoqv24ct0ZcEmXCCqqn8N534zZMCFgA0Ywbc~sh0IO1IxPtOA8zC3mZTgNvImUfR43~Tmq8JuwrouPVY1t33EcSebG~RueHu2xe5Yc~x9QTQzDiFzUJo3047TaO1rq9I~FBp1R6bcpW7MC-A__',
```
Comment (omBratteng) [inline packages/shared/src/lib/image.ts:164]: should this be a cloudinary link instead of figma?

Comment (sshanzel) [reply packages/shared/src/lib/image.ts:164]: Yeah, still don't have it. Let me check with the design team.

Comment (kopancek) [reply packages/shared/src/lib/image.ts:164]: We can fix the image later in a followup PR. I'm making an engineering decision to merge this as is now.

**Pull request merged**